src/main/java/uk/gov/companieshouse/api/interceptor/InternalUserInterceptor.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.companieshouse.api.interceptor;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.lang.StringUtils;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import uk.gov.companieshouse.api.util.security.AuthorisationUtil;
+import uk.gov.companieshouse.api.util.security.EricConstants;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+/**
+ *  Checks that the existing user is an internal user (using the Eric headers). This will be an api-key user.
+ *  All requests to this application must be for internal users
```

Comment:
Can you update this JavaDoc to make it more applicable for a Library (rather than an being part of an application)